airbyte-integrations/connectors/source-instagram/setup.py

```diff
@@ -25,24 +25,28 @@
 
 from setuptools import find_packages, setup
 
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.1",
+    "cached_property~=1.5",
+    "facebook_business~=11.0",
+    "pendulum>=2,<3",
+    "backoff",
```

Comment:
These libraries are installed together with `airbyte-cdk`. Do we need to explicitly specify them?

Reply:
it is better to specify dependencies like this explicitly because CDK might drop this dependency